dbos/_dbos.py

```diff
@@ -231,9 +233,17 @@ def __new__(
                         f"DBOS configured multiple times with conflicting information"
                     )
                 config = _dbos_global_registry.config
+
+            print("calling new setting up the global instance")
```

Comment:
Is it possible to call the DBOS logger for all these statements? So they get exported, etc?